### PR TITLE
doc: add additional info to manual install on osx

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: ansible/MANUAL_STEPS.md [hunk 1]
```diff
@@ -108,8 +108,13 @@ Xcode Command-line tools are not enough to perform a full notarization cycle, fu
 As root:
 
 * Download Xcode: https://developer.apple.com/download/more/ - find non-beta version, open Developer Tools in browser, Networking tab, start download (then cancel), in Networking tab "Copy as cURL" (available in Chrome & FF)
+  * On OSX 11 we currently install 13.2.1
```
Comment (targos) [inline ansible/MANUAL_STEPS.md:111]: We do it on OSX 10.15 as well.

File: ansible/MANUAL_STEPS.md [hunk 2]
```diff
@@ -108,8 +108,13 @@ Xcode Command-line tools are not enough to perform a full notarization cycle, fu
...
 * Download onto release machine using the copied curl command (may need `-o xcode.xip` appended to curl command) to `/tmp`
+  * If you have trouble on the command line, pasting into a shell script file can make pasting/editing to add -o xcode.xip easier
 * Extract: `xip --expand xcode.xip`
+  * This takes a long time since xcode.xip is 7-10G in size depending on version
+  * If you run out of space you can delete `/Users/build/workspace/*` to free up some space. Also make sure that
```
Comment (richardlau) [inline ansible/MANUAL_STEPS.md:116]: FWIW, at least for https://github.com/nodejs/build/issues/3375 most of the reclaimed extra space we needed was from clearing the ccache by running
```console
sudo -s su - iojs
ccache --clear
```
although we did also reclaim a smaller amount of space by removing the workspace.

File: ansible/MANUAL_STEPS.md (hunk 1 above)
Comment (targos) [reply ansible/MANUAL_STEPS.md:111]: Was it reset too? It's annoying as I spent hours installing the same version on all of them

Comment (richardlau) [reply ansible/MANUAL_STEPS.md:111]: I'm not aware of the nearform machines being reconfigured/reset. 

It does look like the orka 10.15 release machine has been reset though and needs a rebuild/restore (currently offline in ci-release) 😞.

Comment (targos) [reply ansible/MANUAL_STEPS.md:111]: I was wrong. I only updated macOS 11 machines: https://github.com/nodejs/build/issues/3311#issuecomment-1529709626

**Review by richardlau — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Pull request merged**